<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ftpWin</class>
 <widget class="QWidget" name="ftpWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>文件传输</string>
  </property>
  <widget class="QListView" name="nodeListView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>350</width>
     <height>500</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>30</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>--- 上传 ---</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>380</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>--- 下载 ---</string>
   </property>
  </widget>
  <widget class="QPushButton" name="upBtn">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>310</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>上传到选定服务器</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>410</y>
     <width>391</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>本地目录:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="downLocalRootEdit"/>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="downSaveRootChooseBtn">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>远端路径:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QLineEdit" name="downRemotePathEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="downloadBtn">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>500</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>下载到本地</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>50</y>
     <width>391</width>
     <height>248</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>本地文件路径:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="upFileChooseBtn">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>远端路径:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="upRemotePathEdit"/>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QListWidget" name="upLocalFileListWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="upClearBtn">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>310</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>清空所选文件</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
